java/spring/pom.xml

```diff
@@ -6,14 +6,14 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>[3.1,3.2)</version>
+		<version>3.1.5</version>
 		<relativePath /> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>benchmark.spring-boot</groupId>
 	<artifactId>benchmark</artifactId>
 	<version>1.0.0</version>
 	<name>benchmark</name>
-	<description>Demo project for Spring Boot</description>
+	<description>Benchmark for Spring Web Mvc (thread per request model)</description>
 
 	<properties>
 		<java.version>17</java.version>
```

Comment:
java 21 ?

Reply:
With Java 21, we can expect significant performance enhancements for this profile with Virtual Threads